workspace.json

```diff
@@ -120,7 +120,8 @@
             "outputPath": "dist/apps/mull-api",
             "main": "apps/mull-api/src/main.ts",
             "tsConfig": "apps/mull-api/tsconfig.app.json",
-            "assets": ["apps/mull-api/src/assets"]
+            "assets": ["apps/mull-api/src/assets"],
+            "showCircularDependencies": false
```

Comment:
Disabling the detection of circular dependencies is probably not a good idea. It actually very useful and will help you from running into circular depend bugs due to the nature of how Node modules work. It's your worst nightmare.

Reply:
I would agree with you, but with something like typeorm, creating circular dependencies is very easy and normal, for example with something like a OneToMany relationship, which requires two classes to import each other. See: 
- nrwl/nx#2975
- nrwl/nx#1667

These warnings aren't helpful in this case, but maybe there's a more granular way to ignore these warning so that other parts still get warnings

Reply:
Yes, it is very normal to have circular dependencies in es modules to a certain degree, but definitely one can run into issues if they aren't cautious. In fact, TypeOrm allows us to use strings for these types of issues as it was pushed below.

https://github.com/typeorm/typeorm/pull/4195

So the question is, would you want to disable circular dependency detection and run into potential issues in the future. Or use something TypeOrm already fixed?

Reply:
Using strings in the annotations won't fix the circular dependency warnings you get, since the imports will stay the same.
I made a branch to showcase this issue `FIX/mull-api-entities-test`, on which you can run the backend to see the issue

Reply:
Let's make this problem of the future, and implement some features. This is no biggie as long as the team knows what circular imports are.